helmfile.d/0200.amphora.yaml

```diff
@@ -31,7 +31,13 @@ releases:
       vcPartner: {{ requiredEnv "AMPHORA_VC_PARTNER_URI" }}
       noSslValidation: {{ env "NO_SSL_VALIDATION" | default false }}
       image:
-        tag: 0.1-SNAPSHOT-1576737535-6-7798b9e
+        registry: {{ env "AMPHORA_IMAGE_REGISTRY" | default "ghcr.io" }}
+        repository: {{ env "AMPHORA_IMAGE_REPOSITORY" | default "carbynestack/amphora-service" }}
+        tag: {{ env "AMPHORA_IMAGE_TAG" | default "latest" }}
```

Comment:
We would like to have deterministic deployments. Please avoid using `latest`. Instead please use fixed version numbers as has been the case before.